### PR TITLE
Add non_cern_machine (Brussels) profile

### PR DESCRIPTION
**Pull request by lpetre-ulb**
## Description

Title speaks for itself.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Motivation and Context

The profile files for non-CERN machine (here Brussels) need to be saved for reference and backup.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: profiles/non_cern_machine/chamberInfo.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
+"""
+``chamberInfo`` --- Information about installed chambers
+--------------------------------------------------------
+"""
```
Comment (bdorney) [inline profiles/non_cern_machine/chamberInfo.py]: I think this should stay with the `gem-plotting-tools` repo.  Later some `user_constants` will be addressed by [this comment](https://github.com/cms-gem-daq-project/gem-plotting-tools/pull/189#issuecomment-472372133)

Comment (bdorney) [reply profiles/non_cern_machine/chamberInfo.py]: so undo this commit if possible (or `git rm` this file).

File: profiles/non_cern_machine/.setup_gemdaq.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```
Comment (bdorney) [inline profiles/non_cern_machine/.setup_gemdaq.sh]: Is it possible the contents of this file could be moved into:

1. `.bash_profile`, and/or
2. `scripts/setup_gemdaq.sh`?

**Review by bdorney — CHANGES_REQUESTED** [review]
Move file contents to existing locations and drop duplicate tracked file.

**Comment by mexanick** [comment]
@lpetre-ulb @bdorney any updates? If this PR is stale, please let me know and it will be closed

**Comment by lpetre-ulb** [comment]
> @lpetre-ulb @bdorney any updates? If this PR is stale, please let me know and it will be closed

Currently, this machine is not fully used. However, the data could be used as basis for another non-CERN machine so I will update the PR today. Thank you for the remainder.

File: profiles/non_cern_machine/chamberInfo.py (hunk 1 above)
Comment (lpetre-ulb) [reply profiles/non_cern_machine/chamberInfo.py]: File removed from commit.

File: profiles/non_cern_machine/.setup_gemdaq.sh (hunk 2 above)
Comment (lpetre-ulb) [reply profiles/non_cern_machine/.setup_gemdaq.sh]: After further investigations, this file was not used. Environment variables were either defined elsewhere or not up-to-date.

**Comment by lpetre-ulb** [comment]
Comments should be addressed. The two files aforementioned are excluded from the commit.

**Review by bdorney — APPROVED** [review]

**Pull request merged**